go/vt/sqlparser/parse_test.go

```diff
@@ -1433,6 +1433,15 @@ var (
 		}, {
 			input:  "analyze table a, b, c",
 			output: "analyze table a, b, c",
+		}, {
+			input:  "prepare stmt from 'select sqrt(pow(?,2) + pow(?,2)) as hypotenuse'",
+			output: "prepare stmt from 'select sqrt(pow(?,2) + pow(?,2)) as hypotenuse'",
+		}, {
+			input:  "execute stmt using @a, @b",
+			output: "execute stmt using @a, @b",
```

Comment:
Would be nice to have a test for `execute stmt` to test that the grammar supports not specifying the optional `using` clause.

Reply:
Would be nice to test with a non-reserved keyword, too, just to assert that code path is working correctly, too.